TreeTracker/UI/Views/Home/Home.storyboard

```diff
@@ -39,6 +40,7 @@
                                                                 <subviews>
                                                                     <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="sapling_Icon" translatesAutoresizingMaskIntoConstraints="NO" id="Jcp-ks-G93">
                                                                         <rect key="frame" x="0.0" y="0.0" width="60" height="75"/>
+                                                                        <color key="tintColor" name="PrimaryGreen"/>
```

Comment:
I'd prefer to keep setting of colours in code - it reduces chance of conflicts and is much more human readable than huge storyboard xml files.

Reply:
gotcha...